webdriver-spec.html

```diff
@@ -7429,21 +7429,42 @@
 <section class=appendix>
 <h2>Security</h2>
 
-<p>A user agent can rely on a command-line flag or a configuration
- option to test whether to enable WebDriver, or alternatively make the
- user agent initiate or confirm the connection through a privileged
- content document or control widget, in case the user agent does not
+<p>A user agent can rely on a command-line flag
+ or a configuration option
+ to test whether to enable WebDriver,
+ or alternatively make the user agent initiate
+ or confirm the connection through
+ a privileged content document or control widget,
+ in case the user agent does not
  directly implement the HTTP endpoints.
 
-<p>To prevent arbitrary machines from connecting and
- creating <a>sessions</a>, it is suggested that only connections from
+<p>It is strongly suggested that user agents
```

Comment:
Good point. Added it to the commit message, but I’m happy to split it out as a separate commit if you think it is best.